fuzz/fuzz_helper/src/lib.rs

```diff
@@ -146,6 +147,22 @@ pub fn process_ctap_any_type(data: &[u8]) -> arbitrary::Result<()> {
     Ok(())
 }
 
+fn setup_storage(unstructured: &mut Unstructured, env: &mut TestEnv) -> arbitrary::Result<()> {
+    if bool::arbitrary(unstructured)? {
+        env.set_enterprise_attestation();
```

Comment:
This is closer to how I perceive this change. A storage setup inside a helper library. My enterprise attestation tests are basically integration tests more than unit tests, and they almost call externally available APIs. (They call `process_make_credential` directly, but that's easy to move up one more indirection to something public.) So maybe `set_enterprise_attestation` could move to `tests/helper` or something?